src/app.rs

```diff
@@ -282,7 +282,21 @@ pub fn build_app() -> App<'static, 'static> {
                 .multiple(true)
                 .hidden_short_help(true)
                 .number_of_values(1),
+        );
+
+    // Make `--one-file-system` available only on Unix and Windows platforms, as per the
+    // restrictions on the corresponding option in the `ignore` crate.
+    // Provide aliases `mount` and `xdev` for people coming from `find`.
+    // It's not pretty, but I'm unaware of a way to make just part of a builder chain conditional
```

Comment:
I think a cleaner way to do this would be to declare `app` as `mut` and then just
```
if cfg!(any(unix, windows)) {
  app = app.arg(…);
}

app
```